main.cpp

```diff
@@ -31,28 +31,240 @@ Purpose:  This project continues developing Project3.
  8) After you finish, click the [run] button.  Clear up any errors or warnings as best you can.
  
  */
-
+#include <iostream>
 /*
  copied UDT 1:
  */
+struct CentralProcessingUnit
+{
+    CentralProcessingUnit(){}
```

Comment:
is this constructor defined inside the class, or outside of the class?
It should be defined outside of the class.
- [ ] revise